query06.sql

```diff
@@ -3,4 +3,10 @@
 */
 
 -- Enter your SQL query here
-select ...
+with indego_trips as (
+  select duration from indego_trips_2019_q2
+  union
+  select duration from indego_trips_2020_q2
+)
+select count(duration < 10)
+from indego_trips
```

Comment:
This won't quite work. It _will_ run, but won't give you what you expect. The `count` function in SQL will count anything that isn't NULL. The expression `duration < 10` is never going to be `null` -- it will either be `true` or `false`. So, this is actually going to count the total number of records across the two tables. You will have to filter the things to count separately, either like:
```sql
select count(*)
from indego_trips
where duration < 10
```

Or, if using Postgres:
```sql
select count(*) filter (where duration < 10)
from indego_trips
```

But the `filter` keyword above isn't available in every database engine (e.g., not in BigQuery).